.lintstagedrc

```diff
@@ -1,7 +1,7 @@
 {
   "*.{js,ts,tsx}": [
     "stylelint",
-    "eslint",
+    "eslint --max-warnings 0 --ignore-pattern !.storybook",
```

Comment:
prevents getting burned by build errors in storybook

Reply:
@jzempel Would it be a good idea to include an `.eslintignore` in the future? The IDE (VS-Code, etc) gets an advantage when using that file with Eslint projects when configuring the ignores globally; devs would be able to see lint errors in the dot directory which is ignored by default, with formatting.

Reply:
ack, for sure. I hadn't seen that `.eslintignore` can be used to config an allowlist.